Mixpanel/MPNetwork.h

```diff
@@ -20,4 +20,12 @@
 
 - (void)updateNetworkActivityIndicator:(BOOL)enabled;
 
+- (NSURLRequest *)requestForEndpoint:(NSString *)endpoint
```

Comment:
If we keep `queryItems` and `andBody` we don't need to pass the http method. Alternatively, we could just keep `byHTTPMethod` and pass an `id` (not ideal, I prefer the first solution).